eGovPDX/portlandor
# POWR-1810 Pattern Lab Breadcrumb Demo and Integration

The Pattern Lab comp for breadcrumb rail is built out and integrated into Drupal.

To Test-
- Go to https://powr-1810-portlandor.pantheonsite.io/
- Navigate through the site. There should be no change in style or functionality of the breadcrumbs.
- Go to https://powr-1810-portlandor.pantheonsite.io/pattern-lab/?p=viewall-components-breadcrumb
- View the demo for main breadcrumb and non-link breadcrumb
- Ensure there is no regression and is properly documented
